web/src/components/navbar/desktop/NavBarDesktopDropDown.tsx

```diff
@@ -60,6 +60,16 @@ export const NavBarDesktopDropDown = (props: Props): ReactElement => {
         transition
         disablePortal={true}
         placement="bottom-end"
+        modifiers={[
+          {
+            name: "flip",
+            enabled: false,
+          },
+          {
+            name: "preventOverflow",
+            enabled: false,
+          },
+        ]}
```

Comment:
All this does is make it so if you try and scroll past the drop down it doesn't change its orientation to try and stay on screen (I was having issues with all the drop downs in the CMS preview covering each other as they tried to stay on screen).

our navbar is sticky so this doesn't affect the main app at all